tools/matchms/matchms_networking.xml

```diff
@@ -22,23 +22,29 @@
     <configfiles>
         <configfile name="matchms_networking_cli">
             python3 ${__tool_directory__}/matchms_networking_wrapper.py \
-            --graph_format "$graph_format" \
-            --identifier "$identifier_key" \
-            --top_n "$top_n" \
-            --max_links "$max_links" \
-            --score_cutoff "$score_cutoff" \
-            --link_method "$link_method" \
+            --score_name '${score_layer_name}_score' \
+            --graph_format '$graph_format' \
+            --identifier '$identifier_key' \
+            --top_n '$top_n' \
+            --max_links '$max_links' \
+            --score_cutoff '$score_cutoff' \
+            --link_method '$link_method' \
             #if $keep_unconnected_nodes
             --keep_unconnected_nodes \
             #end if
-            "$scores" \
-            "$similarity_network_file"
+            '$scores' \
+            '$similarity_network_file'
         </configfile>
     </configfiles>
 
     <inputs>
         <param label="Similarity scores" name="scores" type="data" format="json"
-            help="matchMS similarity-scores file."/>
+            help="matchms similarity-scores file."/>
+        <param label="Score Layer" name="score_layer_name" type="select" help="Metric of the score layer to create a network from.">
+            <expand macro="similarity_metrics"/>
+        </param>
+        <param label="Identifier key" name="identifier_key" type="text"
+            help="Unique metadata identifier of each spectrum from which scores are computed."/>
```

Comment:
This parameter is optional, but running the wrapper without it seems to be always running into error:

```
Traceback (most recent call last):
  File "/m2b/home/trojak/temp/galaxytools/tools/matchms/matchms_networking_wrapper.py", line 38, in <module>
    main(argv=sys.argv[1:])
  File "/m2b/home/trojak/temp/galaxytools/tools/matchms/matchms_networking_wrapper.py", line 31, in main
    network.create_network(scores, args.score_name)
  File "/usr/local/lib/python3.11/site-packages/matchms/networking/SimilarityNetwork.py", line 116, in create_network
    msnet.add_nodes_from(unique_ids)
  File "/usr/local/lib/python3.11/site-packages/networkx/classes/graph.py", line 635, in add_nodes_from
    raise ValueError("None cannot be a node")
ValueError: None cannot be a node
```

with this warning:

```
WARNING:galaxy.model:Datatype class not found for extension 'gml'
```